plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/service/internal/orient/OrientMetadataStore.java

```diff
@@ -113,6 +128,46 @@ public void doStop() throws Exception {
   }
 
   /**
+   * Performs DB backup, during which database is "frozen" for data modification attempts.
```

Comment:
Is the "freezing" automatic as part of the backup process? And what happens to other traffic trying to write data during this window?